tools/shared.py

```diff
@@ -2997,6 +3009,14 @@ class JS(object):
  */
 '''
 
+  closured_emscripten_license = '''\
+/*
+
```

Comment:
Why the extra newline here?  And why not use the ` *` at the start of each line ?

Reply:
This is the exact format of what closure emits. It changes the `/**` to `/*`, removes the `@license` but keeps an empty line for it. That last part does look a little odd, I agree, but closure doens't give us a flag to change that.